packages/api/src/pathfinding.ts

```
@@ -0,0 +1,321 @@
+        }
+      }
+    }
+    
+    // No path found
+    return null;
+  }
+
+  /**
+   * Reconstruct the final path from A* result
+   */
+  private reconstructPath(goalNode: AStarNode, startNode: Node, endNode: Node): RouteResponse {
```

ESLint

[error] 'startNode' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)